dask/highlevelgraph.py

```diff
@@ -705,7 +705,7 @@ def get_all_external_keys(self) -> set:
         except AttributeError:
             keys: set = set()
             for layer in self.layers.values():
-                keys |= layer.get_output_keys()
+                keys.update(layer.get_output_keys())
```

Comment:
Maybe it is worth a comment explaining why the `|=` syntax is not preferable here.

Reply:
Good idea. Done.